### PR TITLE
Added splash screen. #415

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed

#415 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description
Added Splash screen

# Screenshots/casts
https://cdn.discordapp.com/attachments/732444678351355984/764171859989233674/j0pgwlXLq5.mp4

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Review by paulchrisluke — APPROVED** [review]
so pretty <3

**Comment by mort3za** [comment]
@paulchrisluke The splash will be shown only in homepage, is it intended?

**Comment by paulchrisluke** [comment]
anytime someone opens the app please @mort3za

File: src/views/Home.vue [hunk 1]
```diff
@@ -1,47 +1,65 @@
...
-    <template #tertiaryCTA>
-      <p class="text-on-background">
-        Already have an account?
-        <router-link class="" :to="{ name: 'AuthLogin' }">Log In</router-link>
-      </p>
-    </template>
-  </SplashLayout>
+  <div class="w-full h-screen">
+    <transition mode="out-in" v-if="isSplash">
+      <Splash />
```
Comment (mort3za) [inline src/views/Home.vue:4]: Usage of splash component should be in App.vue

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,47 +1,65 @@
...
-      />
-    </template>
-    <template #tertiaryCTA>
-      <p class="text-on-background">
-        Already have an account?
-        <router-link class="" :to="{ name: 'AuthLogin' }">Log In</router-link>
-      </p>
-    </template>
-  </SplashLayout>
+  <div class="w-full h-screen">
```
Comment (mort3za) [inline src/views/Home.vue:2]: We don't use h-screen anywhere. Instead, use h-vh100 please.

**Review by mort3za — CHANGES_REQUESTED** [review]

**Comment by paulchrisluke** [comment]
@mort3za @giteshk11 is currently on foodouken could you finish this one up when you have time?

**Comment by mort3za** [comment]
Sure.

**Pull request closed without merging**